crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -1007,6 +1011,18 @@ impl IndexerStore for PgIndexerStore {
             .serializable()
             .read_write()
             .run(|conn| {
+                // update epoch transaction count
+                let sql = "UPDATE epochs e1
```

Comment:
can we also update some other accumulative metrics like gas etc. as long as checkpoints have it?
```
    pub total_gas_cost: i64,
    pub total_computation_cost: i64,
    pub total_storage_cost: i64,
    pub total_storage_rebate: i64,
```

Reply:
the epoch tx count was a requirement from the FE team..... lets add them if there are requirement, the epoch info response is quite big already....

Reply:
got it, that's fair, even though I feel updating one column is roughly the same as updating several regarding machine cost LOL